README.md

```diff
@@ -1,2 +1,99 @@
-#factory-boy-rest
-###TODO
+factory-boy-rest
+=================
+
+A REST test fixtures replacement for Python.
+
+This is intented to be used on functional tests, to create REST-based entities
+through the same [Factories][#factories] interface provided by
+[Factory Boy][#factory_boy].
+
+[#factories]: http://factoryboy.readthedocs.org/en/latest/reference.html#the-factory-class
+[#factory_boy]: https://github.com/rbarrois/factory_boy
+
+Installation
+------------
+
+```
+pip install factory-rest
+```
+
+
+Development Installation
+------------------------
+
+Create a virtualenv:
+
+```
+mkproject --python=<fullpath_to_python_3> factory-boy-rest
+```
+
+Get the code:
+
+```
+git clone https://github.com/rbarrois/factory_boy.git .
+```
+
+Install it:
+
+```
+python setup.py develop
+```
+
+The last command enter your code in "Development Mode" it creates an
+`egg-link` in your virtualenv's `site-packages` making it available
+on this environment `sys.path`. For more info see [setuptools development-mode]
+(https://pythonhosted.org/setuptools/setuptools.html#development-mode)
+
+
+Development and test dependencies
+---------------------------------
+
+`setup.py` will handle test dependencies, to install development use:
+
+```
+pip install -e .[dev]
+```
+
+
+Tests
+-----
+
+```
+python setup.py test
```

Comment:
Fair enough, adding it... The `python setup.py tests` is great as first shot command because it take care of requirements installation the same for `pytthon setup.py develop`.